helpers/DeliveryHelper.php

```diff
@@ -364,9 +364,19 @@ private static function adjustDeliveryExecutions($deliveryExecutions) {
         $executions = [];
         foreach ($deliveryExecutions as $cachedData) {
 
+            $progressPatern = "/(?<=\- )(.*?)(?= \d)/";
```

Comment:
Having a test case covering this logic wouldn't be out of place

Reply:
It's should be done not in this PR. Also, it's the private static method from helper, and it brings a lot of complexity if we want to create a unit test exactly for that part

Reply:
Do you agree?

Reply:
ok